site/docs/sql/intro.md

```diff
@@ -4,9 +4,12 @@ description: Introduction to Hazelcast Jet SQL
 ---
 
 With Hazelcast Jet you can write SQL to process real-time event streams
-as well as data at rest. The data can be stored inside a Hazelcast
-cluster (including the Jet cluster itself) or in an external storage
-system.
+as well as data at rest. You can pull the data from a Hazelcast cluster
+(including the Jet cluster itself) or from external systems like Kafka
```

Comment:
>including the Jet cluster itself
You can't connect to a remote HZ cluster for now. This suggests as if that was the normal option, but reading locally is also an option.